pandas-stubs/core/resample.pyi

```diff
@@ -181,8 +140,68 @@ class Resampler(BaseGroupBy, Generic[NDFrameT]):
     def last(
         self, numeric_only: bool = ..., min_count: int = ..., *args, **kwargs
     ) -> NDFrameT: ...
-    def mean(self, numeric_only: bool = ..., *args, **kwargs) -> NDFrameT: ...
-    def sem(self, numeric_only: bool = ..., *args, **kwargs) -> NDFrameT: ...
     def median(self, numeric_only: bool = ..., *args, **kwargs) -> NDFrameT: ...
+    def mean(self, numeric_only: bool = ..., *args, **kwargs) -> NDFrameT: ...
+    def std(
+        self, ddof: int = ..., numeric_only: bool = ..., *args, **kwargs
+    ) -> NDFrameT: ...
+    def var(
+        self, ddof: int = ..., numeric_only: bool = ..., *args, **kwargs
+    ) -> NDFrameT: ...
+    def sem(
+        self, ddof: int = ..., numeric_only: bool = ..., *args, **kwargs
+    ) -> NDFrameT: ...
     def ohlc(self, *args, **kwargs) -> DataFrame: ...
-    def nunique(self, *args, **kwargs) -> NDFrameT: ...
+    @overload
+    def nunique(self: Resampler[Series], *args, **kwargs) -> Series[int]: ...
+    @overload
+    def nunique(self: Resampler[DataFrame], *args, **kwargs) -> DataFrame: ...
+    def size(self) -> Series[int]: ...
+    @overload
+    def count(self: Resampler[Series]) -> Series[int]: ...
+    @overload
+    def count(self: Resampler[DataFrame]) -> DataFrame: ...
+    def quantile(
+        self,
+        q: float | list[float] | npt.NDArray[np.float_] | Series[float] = ...,
+        **kwargs,
+    ) -> NDFrameT: ...
+
+# We lie about inheriting from Resampler because at runtime inherits all Resampler
+# attributes via setattr
+class _GroupByMixin(Resampler[NDFrameT]):
+    k: str | list[str] | None
```

Comment:
seems odd that we would export/document something called `k`

Reply:
Good call, this should've been `key`, I'll fix it.